25lessons/script.js

```diff
@@ -0,0 +1,103 @@
+//1
+let number1 = +prompt("Enter first number");
+let number2 = +prompt("Enter second number");
+function number(number1, number2) {
+  if (number1 < number2) {
+    return -1;
+  } else if (number1 > number2) {
+    return 1;
+  } else {
+    return 0;
+  }
+}
+alert(number(number1, number2));
+
+//2
+let factorialNumber = +prompt("Enter number please:");
+function factorialOfNumber(factorialNumber) {
+  let number = 1;
+  for (let i = 1; i <= x; i++) {
+    number *= i;
+  }
+  return number;
+}
+alert(`${factorialOfNumber(factorialNumber)}`);
+
+3;
+
+let numb1 = +prompt("Enter first number please:");
+let numb2 = +prompt("Enter second number please:");
+let numb3 = +prompt("Enter third number please:");
+function allNumberTogether(numb1, numb2, numb3) {
+  let numberSum = String(numb1) + String(numb2) + String(numb3);
+  return numberSum;
+}
```

Comment:
треба тільки перевести це назад в число
```suggestion
function allNumberTogether(numb1, numb2, numb3) {
  let numberSum = String(numb1) + String(numb2) + String(numb3);
  return +numberSum;
}
```